hopsworks-IT/src/test/ruby/spec/kafka_acl_spec.rb

```diff
@@ -0,0 +1,501 @@
+=begin
+ This file is part of Hopsworks
+ Copyright (C) 2019, Logical Clocks AB. All rights reserved
+
+ Hopsworks is free software: you can redistribute it and/or modify it under the terms of
+ the GNU Affero General Public License as published by the Free Software Foundation,
+ either version 3 of the License, or (at your option) any later version.
+
+ Hopsworks is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+ PURPOSE.  See the GNU Affero General Public License for more details.
+
+ You should have received a copy of the GNU Affero General Public License along with this program.
+ If not, see <https://www.gnu.org/licenses/>.
+=end
+
+require 'json'
+
+describe "On #{ENV['OS']}" do
+  describe 'kafka acl' do
+    after (:all) { clean_projects }
+
+    describe 'basic acl operations' do
+      context 'with valid project with two users, kafka service enabled, and a kafka schema' do
+        before :all do
+          with_valid_project
+          project = get_project
+          new_member = create_user[:email]
+          post "#{ENV['HOPSWORKS_API']}/project/#{@project[:id]}/projectMembers", {projectTeam: [{projectTeamPK: {projectId: @project[:id],teamMember: new_member},teamRole: "Data scientist"}]}
+          expect_json(successMessage: "One member added successfully")
+          expect_status(200)
+          with_kafka_schema(project.id)
+          with_kafka_topic(project.id)
+        end
+        after :all do
+          project = get_project
+          clean_topics(project.id) 
```

Comment:
@kai-chi shall we clean the schemas as well or will it be done with the next PR?

Reply:
good catch. it could be added here but since schemas are gonna go through a huge refactoring in the next PR I would add it there. is that acceptable?

Reply:
Ok let's do that